src/router.tsx

```diff
@@ -33,7 +33,7 @@ import UpdateNodePage from './steps/updateNode';
 import SetupNodePage from './steps/setupYourNode';
 import AddedToWhitelist from './steps/addedToWhitelist';
 import JoinWaitlistPage from './steps/joinWaitlist';
-import WhatYouWillNeedPage from './steps/whatYouWillNeed';
+import WhatYouWillNeedPage from './steps/step0_whatYouWillNeed';
```

Comment:
this comment is not specific to this exact file but the naming convention used in this file goes against what we have used in the rest of the repo so maybe we can have a folder step0 and inside that folder we can have that file

Reply:
folder is ok, @diegoalzate Diego please create a task to do on the topic, tbd when all steps are working

Reply:
ok